api-response.go

```diff
@@ -218,6 +218,27 @@ type CompleteMultipartUploadResponse struct {
 	ETag     string
 }
 
+// DeletedObject - deleted object.
+type DeletedObject struct {
```

Comment:
Same here. This type is not necessary or at least it can reuse the objectid from request type.